pkg/streamingpromql/benchmarks/benchmarks.go

```diff
@@ -130,6 +130,9 @@ func TestCases(metricSizes []int) []BenchCase {
 			Expr:  "a_X - b_X",
 			Steps: 10000,
 		},
+		{
+			Expr: "nh_X / a_X",
```

Comment:
Would be good to add a benchmark for the case where there's native histograms on both sides